eng/scripts/generate_from_source_pom.py

```diff
@@ -120,8 +120,31 @@ def create_from_source_pom(project_list: str, set_skip_linting_projects: str, ma
 
     # output the ServiceDirectories environment variable
     service_dirs = list(sorted(service_directories))
-    print('setting env variable ServiceDirectories = {}'.format(service_dirs))
-    print('##vso[task.setvariable variable=ServiceDirectories;]{}'.format(json.dumps(service_dirs)))
+    # JRS need to create service_directory_full_path which will add root_path
```

Comment:
Is this a leftover comment you still need to follow-up on?

Reply:
It's a leftover comment. If not for needing to test the release portion, I wouldn't have checked this PR in but since the pipeline yml only existed in this PR, I had a chicken and egg thing going. Anything with my initials will be removed with the next PR.